gazebo_ros_control/CMakeLists.txt

```diff
@@ -31,9 +33,11 @@ catkin_package(
     joint_limits_interface
     urdf
     angles
-  INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME} default_robot_hw_sim
-  DEPENDS gazebo
```

Comment:
That's a really good answer, would be great if it was added to the catkin documentation. I added a link from the wiki to that answer, for now.

Seems I'll have the same pthread linking issues now, I'll work on it soon.